CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+## Unreleased
```

Comment:
These need to go into CHANGELOG_PENDING rather than CHANGELOG as we automate the release notes from that file